bench/wtperf/wtperf.c

```diff
@@ -1955,9 +1955,9 @@ start_all_runs(CONFIG *cfg)
 
 	home_len = strlen(cfg->home);
 	cmd_len = (home_len * 2) + 30; /* Add some slop. */
-	cmd_buf = dcalloc(cmd_len, 1);
+	cmd_buf = dmalloc(cmd_len);
 	for (i = 0; i < cfg->database_count; i++) {
-		next_cfg = dcalloc(1, sizeof(CONFIG));
+		next_cfg = dmalloc(sizeof(CONFIG));
```

Comment:
@keithbostic, is this safe? Looks like `next_cfg` as used in `config_assign` needs to be cleared, else `config_free` will access `config_head` and other fields and be very confused.